.typo-ci.yml

```diff
@@ -0,0 +1,10 @@
+# Any typos we should ignore?
+excluded_words:
+  - Devnol
+  - Rexogamer
+  - ttsbotsnapps
+  - xaml
+  - AboutRedirButtonMyEmail
+  - runas
+  - QuantumlyTangled
+  - updatereleasezip
```

Comment:
these seem pretty pointless and irrelevant imo. Also add Midou.

Reply:
Once again, a copypaste from [Fixdows](https://github.com/Odyssey346/Fixdows)

Reply:
Add Miicord too (just in case)

Reply:
added miicord ✔️